tests/unit/replace-widget.spec.ts

```diff
@@ -30,24 +30,46 @@ describe('Widget ReplaceWidget', () => {
       store: emptyStore,
       localVue,
       sync: false,
-      data: () => {
-        return { toReplace: ['foo', 'bar'] };
+      propsData: {
+        value: ['foo', 'bar'],
       },
     });
     const widgetWrappers = wrapper.findAll('inputtextwidget-stub');
     expect(widgetWrappers.at(0).props().value).toEqual('foo');
     expect(widgetWrappers.at(1).props().value).toEqual('bar');
   });
 
-  it('should emit "input" event on "toReplace" update', async () => {
+  it('should emit "input" event with correct updated values when input valueToReplace is updated', async () => {
     const wrapper = shallowMount(ReplaceWidget, {
+      propsData: {
+        value: ['yolo', 'bim'],
+      },
+      store: emptyStore,
+      localVue,
+      sync: false,
+    });
+    await localVue.nextTick();
```

Comment:
Not sure this nextTick would be necessary if we avoid sending the first unnecessary input